src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing Cloud Credentials, a central location to manage your authentication with AWS and GCP  <!-- 8550 -->
+- Enabling Okteto Insights for all users of Okteto <!-- 8627 -->
+- Added configurable buildkit network mode  <!-- 8486 -->
+- Init dynamic config  <!-- https://github.com/okteto/app/pull/8475 -->
+- Added CLI, Chart, and Okteto App version to our API headers  <!-- 8545 -->
+- Support to enable/disable resource quotas for individual Namespaces  <!-- 8604 -->
+- Allow override of installer security context  <!-- https://github.com/okteto/app/pull/8608 -->
+
+### Improvements
+
+- Show more actionable feedback upon "failed to deploy okteto pipeline"…  <!-- 8521 -->
+- You can now list your Github App installations under Settings → Integrations  <!-- 8463 -->
+- Implemented a number of UI updates to ensure your Okteto experience stays smooth <!-- 8600, 8623, 8534, 8626 -->
+
+### Bug Fixes
+
+- Fixed wrong data parsing error from unknown git urls  <!-- 8504 -->
+- Fixed collapsed content not being accessible   <!-- 8522 -->
+- Replaced `Input` with `FormField` in the login view  <!-- 8510 -->
```

Comment:
Not sure this is relevant for users. The one about the input seems more internal.

The first is hard to know if it is affecting something customer facing or not. It doesn't specify where this content wasn't being accessible. Is that relevant for users?

Reply:
@viddo Do you think it's worth mentioning?

Reply:
Yes, but not the way they are currently worded. The texts appears to be verbatim from the PR titles. I'd suggest to reword these based on the PR descriptions, something on the lines of:

```suggestion
- Fixed issue where interactive but hidden elements, such as links in closed deploy log stage, were wrongly accessible via keyboard navigation and screen readers <!-- 8522 -->
- Fixed issue preventing password managers from filling in the login token <!-- 8510 -->
```

Similarly, I think there's more UI bugfixes and improvements we done since time that are worthwhile to include in the release notes. I'll review internal changelog and add some more suggestions here & there probably.

---

BTW, as an aside, I think we could improve how we articulate PR titles/desc and auto-generate the release notes to avoid having to go back on so many items in this kind of docs PR. Let's talk about it in a retrospective and/or after this release to improve for next time?